src/parser/transform/constraint/transform_constraint.cpp

```diff
@@ -70,12 +70,24 @@ unique_ptr<Constraint> Transformer::TransformConstraint(duckdb_libpgquery::PGLis
 		return make_unique<UniqueConstraint>(index, false);
 	case duckdb_libpgquery::PG_CONSTR_NULL:
 		return nullptr;
+	case duckdb_libpgquery::PG_CONSTR_GENERATED_VIRTUAL: {
+		if (column.DefaultValue()) {
+			throw InvalidInputException("DEFAULT constraint on GENERATED column \"%s\" is not allowed", column.Name());
+		}
+		column.SetGeneratedExpression(TransformExpression(constraint->raw_expr));
+		return nullptr;
+	}
+	case duckdb_libpgquery::PG_CONSTR_GENERATED_STORED:
+		throw InvalidInputException("Can not create a STORED generated column!");
 	case duckdb_libpgquery::PG_CONSTR_DEFAULT:
-		column.default_value = TransformExpression(constraint->raw_expr);
+		if (column.Generated()) {
+			throw InvalidInputException("DEFAULT constraint on GENERATED column \"%s\" is not allowed", column.Name());
```

Comment:
This line is never run

Reply:
Line is removed, generated constraint has to be provided first so this never happens